lib/ansible/modules/cloud/vmware/vmware_vsan_health_facts.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: vmware_vsan_health_facts
```

Comment:
All facts modules are converted into info, so I would recommend you to rename this to `vmware_vsan_health_info`. Other module rename is happening [here](https://github.com/ansible/ansible/pull/57474)